sdcm/sct_config.py

```diff
@@ -626,6 +626,9 @@ class SCTConfiguration(dict):
         dict(name="security_group_ids", env="SCT_SECURITY_GROUP_IDS", type=str_or_list,
              help="AWS security groups ids to use"),
 
+        dict(name="use_placement_group", env="SCT_USE_PLACEMENT_GROUP", type=str,
+             help="if true, create placement group for test case for low-latency network performance achievement"),
```

Comment:
there are different kinds of placement groups, mention the strategy here.

Reply:
comment was updated